sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -57,18 +56,20 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
 
       val maybeAnalyzedPredicate = query.queryExecution.optimizedPlan.collect {
         case PhysicalOperation(_, filters, LogicalRelation(_: ParquetRelation)) => filters
-      }.flatten.reduceOption(_ && _)
+      }.flatten
+      assert(maybeAnalyzedPredicate.nonEmpty)
```

Comment:
Please rename `maybeAnalyzedPredicate` to `analyzedPredicates` since it's now a `Seq[Expression]`. Prefix `maybe` is for `Option[T]` by convention.